packages/fxa-auth-server/test/oauth/db/index.js

```diff
@@ -574,14 +581,41 @@ describe('db', function () {
     });
   });
 
-  describe('getLock', function () {
-    it('should return an acquired status', function () {
-      const lockName = randomString(10);
-      return db.getLock(lockName, 3).then(function (result) {
-        assert.ok(result);
-        assert.ok('acquired' in result);
-        assert.ok(result.acquired === 1);
-      });
+  describe('pruneAuthorizationCodes', () => {
```

Comment:
All the existing tests here perform actual DB operation, so I'm trying to be consistent.  The only wrinkle is that I need to write the code inserting query to add the codes to be pruned since that query isn't part of the service.